iroh-net/src/relay/http/server.rs

```diff
@@ -108,13 +146,15 @@ pub struct TlsConfig {
     pub acceptor: TlsAcceptor,
 }
 
-/// Build a Relay Server that communicates over HTTP or HTTPS, on a given address.
+/// Builder for the Relay HTTP Server.
 ///
-/// Defaults to handling relay requests on the "/derp" endpoint.
+/// Defaults to handling relay requests on the "/derp" endpoint.  Other HTTP endpoints can
```

Comment:
we should transition this name as well, @flub can you make an issue to migrate this to sth like `/relay`?

Reply:
https://github.com/n0-computer/iroh/issues/2378